sdk/storage/storage-blob/src/Clients.ts

```diff
@@ -1256,6 +1256,11 @@ export class BlobClient extends StorageClient {
    * for the blob. It does not return the content of the blob.
    * @see https://docs.microsoft.com/en-us/rest/api/storageservices/get-blob-properties
    *
+   * WARNING: If the option to include metadata is specified (as in `getProperties({ include: ["metadata"] })`),
```

Comment:
I don't think `getProperties()` methods take any `include` option. So they always return the user-defined metadata. The option is used only for listing methods.

Reply:
right, I misread the description in the issue. It's the list methods that take this option and getProperties always produces the metadata. The difference between `includeMetadata` and `include: ['metadata']` caught me here.